Example/GiniVision/SettingsViewController.swift

```diff
@@ -49,11 +50,19 @@ final class SettingsViewController: UIViewController {
         giniConfiguration.qrCodeScanningEnabled = sender.isOn
         delegate?.settings(settingViewController: self, didChangeConfiguration: giniConfiguration)
     }
+    
+    @IBAction func multipageSwitch(_ sender: UISwitch) {
+        giniConfiguration.multipageEnabled = sender.isOn
+        delegate?.settings(settingViewController: self, didChangeConfiguration: giniConfiguration)
+
+    }
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         openWithSwitch.setOn(giniConfiguration.openWithEnabled, animated: true)
         qrCodeScanningSwitch.setOn(giniConfiguration.qrCodeScanningEnabled, animated: true)
-
+        multipageSwitch.setOn(giniConfiguration.multipageEnabled, animated: true)
```

Comment:
It makes no sense to have it animated in the `viewDidLoad` method.

Reply:
True 👍